pkg/config/config_template.yaml

```
@@ -1193,6 +1193,14 @@ api_key:
   #
   # file_wildcard_selection_mode: `by_name`
 
+  ## @param max_message_size_bytes - integer - optional - default: 256000
+  ## @env DD_MAX_MESSAGE_SIZE_BYTES - integer - optional - default : 256000
```

```suggestion
  ## @env DD_LOGS_CONFIG_MAX_MESSAGE_SIZE_BYTES - integer - optional - default : 256000
```
I believe this is the correct env var